packages/next/next-server/server/config.ts

```diff
@@ -23,6 +23,7 @@ const defaultConfig: { [key: string]: any } = {
   target: 'server',
   poweredByHeader: true,
   compress: true,
+  images: { hosts: { default: { path: 'defaultconfig' } } },
```

Comment:
There needs to be some kind of validation on the `images` property otherwise it's going to be easy to put in the wrong values

Reply:
do you think the checks at https://github.com/vercel/next.js/pull/17343/files#diff-4ec59a55144129a45f46da71c0777df9R232 are enough for this? Those should break the build if they pass a image config that doesn't have a default host or that provides hosts that don't have the necessary `path` property.